src/cart/crt_corpc.c

```diff
@@ -42,14 +42,15 @@ crt_corpc_info_init(struct crt_rpc_priv *rpc_priv,
 	D_ASSERT(rpc_priv != NULL);
 	D_ASSERT(grp_priv != NULL);
 
-	D_ALLOC_PTR(co_info);
+	co_info = d_mm_alloc(sizeof(*co_info));
 	if (co_info == NULL)
 		D_GOTO(out, rc = -DER_NOMEM);
+	memset(co_info, 0, sizeof(*co_info));
```

Comment:
Most, but not all invocations call memset(), and there's another call to sizeof here (on a small size, which implies the cost is going to be in terms of function entry/exit rather than memory bandwidth), should we have the new macro take a flag to return zeroed memory or not?